bedrock/firefox/templates/firefox/channel/android.html

```diff
@@ -31,75 +32,97 @@
-      </ul>
-    </div>
-  </footer>
 </section>
 
-<section id="nightly" class="channel">
-  <div class="container">
-    <header>
-      <h2>{{_('Nightly')}}</h2>
-      {% if l10n_has_tag('android_nightly_copy') %}
-        <p>{{_('Check out new Android features in their earliest stages. Enjoy at your own risk.')}}</p>
-      {% else %}
-        {# Use the existing Nightly desktop copy as a fallback #}
-        <p>{{_('Test brand new features daily (or… nightly). Enjoy at your own risk.')}}</p>
-      {% endif %}
-    </header>
-    {{ download_firefox('nightly', platform='android', alt_copy=_('Download'), dom_id="android-nightly-download") }}
-    <p class="notice">
-        {% if l10n_has_tag('privacy_201806') %}
-        {%- trans link=url('privacy.notices.firefox') + '#pre-release' -%}
-          Nightly is an unstable testing and development platform. By default, Nightly sends data to Mozilla
-          — and sometimes our partners — to help us handle problems and try ideas.
-          <a href="{{ link }}">Learn what is shared</a>.
-        {%- endtrans -%}
-      {% else %}
-        {{_('Firefox Nightly automatically sends feedback to Mozilla.')}}
-        <a href="{{ url('privacy.notices.firefox') + '#pre-release' }}">{{_('Learn more')}}</a>
-      {% endif %}
-    </p>
-  </div>
-  <footer>
-    <div class="container">
-      <ul>
-        <li><a rel="external" href="https://blog.nightly.mozilla.org/">{{_('Nightly Blog')}}</a></li>
-        <li><a href="{{ firefox_url('android', 'all', 'nightly') }}">{{_('All Languages and Builds')}}</a></li>
-      </ul>
+<section id="nightly">
+  <section class="mzp-c-call-out-compact mzp-t-firefox">
+    <img class="logo" width="120" height="120" alt="" src="{{ static('protocol/img/logos/firefox/browser/nightly/logo.svg') }}" />
+    <div class="mzp-l-content">
+      <div class="mzp-c-call-out-content">
+        <div class="mzp-c-call-out-container">
+          <h2 class="mzp-c-call-out-title">{{ _('Nightly') }}</h2>
+          <p class="mzp-c-call-out-desc">
+            {% if l10n_has_tag('android_nightly_copy') %}
+              {{_('Check out new Android features in their earliest stages. Enjoy at your own risk.')}}
+            {% else %}
+              {{_('Test brand new features daily (or… nightly). Enjoy at your own risk.')}}
+            {% endif %}
+          </p>
+          <ul>
+            <li><a rel="external" href="https://blog.nightly.mozilla.org/">{{_('Nightly Blog')}}</a></li>
+            <li><a href="{{ firefox_url('android', 'all', 'nightly') }}">{{_('All Languages and Builds')}}</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="mzp-c-call-out-cta">
+        <div class="mzp-c-call-out-cta-container">
+          {{ download_firefox('nightly', platform='android', alt_copy=_('Download'), dom_id="android-nightly-download") }}
+        </div>
+      </div>
     </div>
-  </footer>
+  </section>
+
+  <div class="mzp-l-notes">
```

Comment:
Nit: avoid adding mzp namespace to things that aren't implemented in Protocol?
```suggestion
  <div class="l-notes">
```